docs/newContributors.md

```diff
@@ -0,0 +1,86 @@
+<h1 align="center">Hello Peeps 👋</h1>
+<h2 align="center">Welcome to the <a href="https://github.com/siv2r/kidney-exchange">kidney-exchange</a> Repo 🎉</h2>
+
+<h2 align="center">Here, is the Complete Guide to Contribute to this Repo.</h2>
+
+
+
+ 1.  Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+      > **Open Source is more about Collaboration.** 
+
+    
+
+ 2.  So to start Contributing to our Project you can solve existing issues or if you have found any bugs or want to add a new feature to Project you can create New Issue, that's purely your choice.
+
+
+
+  ![](../images/Issues.png)
+
+
+
+ 3.  Project Maintainers and Mentors will guide you with the issue and incase of any problem you face, you can take help from the community in the discord server.
+
+
+ 4.  So, Now you have an issue to work upon. So first you need is to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way:-
+
+
+
+  ![](../images/Fork.png)
+
+
+ 5.  Clone the Repo you have forked:-
+
+
+  
+      - Copy the link of the forked Repo.
+      
+      - Go to the desired location in the terminal, where you want to place the project.
+      
+      - Run command ``git clone link_of_forked_repo``.
```

Comment:
you can use `git clone https://github.com/<github username>/kidney-exchange.git` instead